lib/cloudfront/signer.js

```diff
@@ -157,7 +157,8 @@ AWS.CloudFront.Signer = inherit({
      *
      * @param options [Object]          The options to create a signed URL.
      * @option options url [String]     The URL to which the signature will grant
-     *                                  access. Required.
+     *                                  access. Any query params included with 
```

Comment:
```suggestion
     *                                  access. Any query params included with
```
Looks like a trailing space is causing the linter to complain.